### ISSUE TITLE
Consider Paypal donate

### ISSUE THREAD

**Issue by midspace** [midspace/Space-Engineers-Admin-script-mod#61]
Should we add a Paypal donate button?

Either to here on Giuthub, or Steam.

I have no idea if we can legally add them to either.
I think Valve recently went through a purge of donate buttons after the Skyrim paid mod fiasco.

I have a paypal account, but then I have to consider how to break up any payments.
Unless paypal can do that already.
Unless there are other options.

**Comment by Spcemarine** [comment]
If it is possible, I have no objections. But I wouldn't put it here because less people get here.
I don't really know if donations on steam are legal but I think there are many who add a button to their mods and valve does nothing against it... And the thing with Skyrim was to force the modders to use the pay option. In any case KSWH has to agree that we are allowed to get donations, do you know if they officially allowed it?

Google says that somehow it is possible to have more than one receiver but I'm not sure if this works with donation buttons: https://developer.paypal.com/docs/classic/adaptive-payments/integration-guide/APIntro/

**Comment by jpcsupplies** [comment]
A "tip" button would be safer, having used donate buttons before the politics behind it usually makes paypal review your account and freeze it until you complete a corporate, charity and taxation entity qualification check process with them.  While a tip button is basically a pay for service type button, which is apparently just fine; until you start getting too many payments that is..

Steam also has its own version of a monetization system, you just fill out a form and then valve have no problem with you monetizing.. assuming the politics hasn't had the game you want to monetize blacklisted (ie skyrim)   I accidentally went through the process when a rather misleading screen popped up in steam last time I tried to add a new workshop item; basically you just fill out a form saying nope not in the USA, and your credentials.  It generates some W something US IRS form that excempts you and steam from the need to report earnings to the USA(if you are not in the USA nor a citizen), Not tried to monetize anything tho, but I am allowed to apparently...

Other options are paymate an Australian competitor to paypal that dont care what you get money for as long as they get their fee.

**Comment by zrisher** [comment]
Yeah in the US you can't ask for a "donations" unless you're a 501(c) tax-empty charitable organization. Paypal has to do this because otherwise some people would point to the transaction on their taxes and say "I donated! Deduct it from my AGI!"

**Comment by midspace** [comment]
Considering my current paypal account is under paypal.com.au, its legal status falls under Australian jurisdiction. Paypal also requires a business account if paying multiple people, and considers them employees.
Because of that and a lot of the legal mumbo jumbo, I've been very hesitant to do anything in that regard to adding any sort of donation button.

The way it's going, the Australian Government may start demanding GST payments from PayPal Australia in the near future, which may mean a lot of red tape and paper work in real life for what might be considered loose change.

For the moment I'm happy to just leave it as it is currently, unless @Spcemarine has a better idea.

**Comment by jpcsupplies** [comment]
My account is Australian as well; the ATO are behind a lot of paypal politics.. at least that is what they told me last time my account froze.  Just avoid using their donate button making tool and stick to their web payments system, and dont word it as a donation.
Or there is paymate which is Australian and has less paperwork, but higher per transaction fees.
Or go totally off the map and accept crypto like bitcoin.

**Comment by Spcemarine** [comment]
I agree with midspace. If there is too much paperwork let's leave it like that. I mean it's not like we are doing this to get something...
